dags/socorro_import.py

```diff
@@ -18,39 +38,154 @@
+s3_to_gcs = S3ToGoogleCloudStorageTransferOperator(
+    task_id='s3_to_gcs',
+    s3_bucket='crashstats-telemetry-crashes-prod-us-west-2',
+    gcs_bucket=gcs_data_bucket,
+    description='socorro crash report copy from s3 to gcs',
+    aws_conn_id=read_aws_conn_id,
+    gcp_conn_id=gcp_conn_id,
+    project_id=connection.project_id,
+    object_conditions=gcstj_object_conditions,
+    transfer_options=gcstj_transfer_options,
+    dag=dag
+)
+
+# Spark job reads gcs json and writes gcs parquet
+crash_report_parquet = SubDagOperator(
     task_id="crash_report_parquet",
-    job_name="Socorro Crash Reports Parquet",
-    execution_timeout=timedelta(hours=4),
-    instance_count=10,
-    env={"date": "{{ ds_nodash }}"},
-    uri="https://raw.githubusercontent.com/mozilla-services/data-pipeline/master/reports/socorro_import/ImportCrashData.ipynb",
-    output_visibility="public",
+    dag=dag,
+    subdag = moz_dataproc_pyspark_runner(
+        parent_dag_name=dag.dag_id,
+        dag_name='crash_report_parquet',
+        default_args=default_args,
+        cluster_name=cluster_name,
+        job_name="Socorro_Crash_Reports_to_Parquet",
+        # TODO - CI/CD for gcs and repo sync for artifact bucket jobs and gcp bootstrap folders
+        python_driver_code="gs://moz-fx-data-prod-airflow-artifacts/jobs/socorro_import_crash_data.py",
+        py_args=[
+            "-d", "{{ ds_nodash }}",
+            "--source-gcs-path", "gs://{}/v1/crash_report".format(gcs_data_bucket),
+            "--dest-gcs-path", "gs://{}/{}".format(gcs_data_bucket, dataset)
+        ],
+        idle_delete_ttl='14400',
+        num_workers=8,
+        worker_machine_type='n1-standard-8',
+        aws_conn_id=read_aws_conn_id,
+        gcp_conn_id=gcp_conn_id)
+)
+
+# TODO - change me later to remove the _2
+bq_gcp_conn_id= 'google_cloud_derived_datasets_2'
+bq_connection = GoogleCloudBaseHook(gcp_conn_id=bq_gcp_conn_id)
+gke_location="us-central1-a"
+gke_cluster_name="bq-load-gke-1"
+# TODO - change this to just s3://telemetry-parquet when ready
+dest_s3_key="s3://telemetry-parquet/socorro_crash_hwoo"
+
+# We don't use -m for parallel rsync because I ran into 503 while issues using it.
+gsutil_args = [
+    'gsutil', 'rsync', '-r',
+    'gs://{}/{}'.format(gcs_data_bucket, objects_prefix),
+    '{}/{}'.format(dest_s3_key, objects_prefix)
+]
+
+# Copy parquet data from gcs to s3
+gcs_to_s3 = GKEPodOperator(
+    task_id='gcs_to_s3',
+    gcp_conn_id=bq_gcp_conn_id,
+    project_id=bq_connection.project_id,
+    location=gke_location,
+    cluster_name=gke_cluster_name,
+    name='copy-parquet-data-back-to-s3',
+    namespace='default',
+    image="google/cloud-sdk:242.0.0-alpine",
+    arguments=gsutil_args,
+    env_vars={
+        "AWS_ACCESS_KEY_ID": "{{ var.value.socorro_access_key_id }}",
+        "AWS_SECRET_ACCESS_KEY": "{{ var.value.socorro_secret_access_key }}"
+    },
+    dag=dag,
+)
+
+# We dont use load_to_bigquery since our source data is s3 json.
```

Comment:
This is unclear to me, I'd write:
```suggestion
# We don't use load_to_bigquery since our source data is on GCS.
```
instead. Unless by `source data` you mean original Socorro files, but that's not relevant to BQ load?